de.willuhn.Jameica.yaml

```diff
@@ -1,15 +1,18 @@
 app-id: de.willuhn.Jameica
 default-branch: stable
 runtime: org.gnome.Platform
-runtime-version: '42'
+runtime-version: '43'
 sdk: org.gnome.Sdk
 sdk-extensions:
   - org.freedesktop.Sdk.Extension.openjdk11
 command: jameica-wrapper.sh
 finish-args:
   - --socket=x11
   - --socket=wayland
-  - --socket=session-bus
+  # - --socket=session-bus
```

Comment:
please remove the commented line - it'll still be available in git history if needed